vars/githubscm.groovy

```diff
@@ -165,6 +165,37 @@ Tag Message: ${tagMessage}
 """
 }
 
+def pushRemoteTag(String remote, String tagName, String credentialsId = 'kie-ci') {
+    pushObject(remote, "--tags ${tagName}", credentialsId)
+    println "[INFO] Pushed remote tag ${tagName}."
+}
+
+boolean isTagExist(String remote, String tagName) {
+    sh "git fetch ${remote} --tags"
+    return sh(returnStatus: true, script: "git rev-parse ${tagName}") == 0
+}
+
+void removeLocalTag(String tagName) {
+    sh "git tag -d ${tagName}"
+    println "[INFO] Deleted tag ${tagName}."
+}
+
+def removeRemoteTag(String remote, String tagName, String credentialsId = 'kie-ci') {
```

Comment:
Should you have an admin rights to remove the tag?

Reply:
What it the tag does not exist?

Reply:
> `Should you have an admin rights to remove the tag?`
I guess so
> `What if the tag does not exist? Shouldn't? we check it here as well?`
This would be the responsibility of the pipeline

Reply:
> > `Should you have an admin rights to remove the tag?`
> 
> I guess so
> 
> > `What if the tag does not exist? Shouldn't? we check it here as well?`
> 
> This would be the responsibility of the pipeline

Then maybe it worth to comment that?

Reply:
@dupliaka comments added